app/views/brexit_checker/results.html.erb

```diff
@@ -37,6 +37,18 @@
 <% end %>
 
 <div class="govuk-width-container brexit-checker-results-page">
+  <% if show_confirmation_reminder? %>
+    <%= render "govuk_publishing_components/components/notice", {
+      margin_bottom: 0
+    } do %>
+      <p class="govuk-body">
+        <%= sanitize(t("brexit_checker.results.accounts.confirm.intro.#{confirmation_banner_prompt_type}")) %>
+      </p>
+      <p class="govuk-body govuk-!-margin-bottom-0">
+        <a href="<%= "#{Plek.find('account-manager')}/account/confirmation/new" %>" class="govuk-link"><%= t("brexit_checker.results.accounts.confirm.link_text") %></a>
```

Comment:
We'll need to add the Plek env vars for this

Reply:
This was done, will be resolving this